src/libstd/net/ip.rs

```diff
@@ -231,6 +234,8 @@ impl Ipv4Addr {
     /// - the broadcast address (255.255.255.255/32)
     /// - test addresses used for documentation (192.0.2.0/24, 198.51.100.0/24 and 203.0.113.0/24)
     /// - the unspecified address (0.0.0.0)
+    ///
+    /// [ipv4-sr]: http://goo.gl/RaZ7lg
```

Comment:
To rephrase my previous question: Why don't we just put the target URL here instead of the shortened URL? Not a big deal though, just curious